### ISSUE TITLE
plantum on macOS not working

### ISSUE THREAD

**Issue by StefanoCecere** [shd101wyy/vscode-markdown-preview-enhanced#395]
my Plantum doesn't work anymore... the preview page is always blank
i could find any error in the console and system log
how could i debug?

what is the exact command that is used to launch the plantum.jar?

**Comment by shd101wyy** [comment]
Could you please show me a screenshot?
Also try close the preview then reopen it.

**Comment by StefanoCecere** [comment]
tried everything.. but always "blank".. Dot works

<img width="825" alt="Screenshot 2020-07-24 at 17 19 27" src="https://user-images.githubusercontent.com/640536/88407068-d98d5a80-cdd1-11ea-9603-41693c91be44.png">


<img width="1017" alt="Screenshot 2020-07-24 at 17 19 57" src="https://user-images.githubusercontent.com/640536/88407112-ed38c100-cdd1-11ea-9c2e-cd92fc5100d8.png">

**Comment by shd101wyy** [comment]
Is the console displaying any error?

**Comment by StefanoCecere** [comment]
no, i couldn't find any trace of errors in the Electron Console nor System Logs
that's why i asked how the plugin was calling the plantuml.. maybe i could debug via Terminal!

**Comment by shd101wyy** [comment]
How about reinstalling the extension? Which version of VSCode are you using?

**Comment by tmori3y2** [comment]
Did you install Java?
